hy/compiler.py

```diff
@@ -2,6 +2,7 @@
 # Copyright 2018 the authors.
```

Comment:
The refactoring diffs in this file are big enough that they should get their own PR. I'm not sure they're all improvements, though. For example, you moved `_loopers` away from where it's used.

Reply:
Removed them